prog/kubernetes/kubernetes_nodepool_nexus.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class Prog::Kubernetes::KubernetesNodepoolNexus < Prog::Base
+  subject_is :kubernetes_nodepool
+
+  def self.assemble(name:, node_count:, kubernetes_cluster_id:)
+    DB.transaction do
+      unless KubernetesCluster[kubernetes_cluster_id]
+        fail "No existing cluster"
+      end
+
+      kn = KubernetesNodepool.create(
+        name: name,
+        node_count: node_count,
+        kubernetes_cluster_id: kubernetes_cluster_id
+      )
```

Comment:
nit:
```suggestion
      kn = KubernetesNodepool.create(name:, node_count:, kubernetes_cluster_id:)
```

Reply:
This is not one of my favorite features, but not opposed to using it.